text/0225-configuration.md

```diff
@@ -0,0 +1,375 @@
+
+The method returns a [Configuration model](#configuration-model) that has been validated. This API *MAY* return an error or raise an exception, whichever is idiomatic to the implementation for the following reasons:
+
+* file doesn't exist or is invalid
+* configuration parsed is invalid according to schema
+
+#### Python Parse example
+
+```python
+
+filepath = "./config.yaml"
+
+
+try:
+  cfg = opentelemetry.Parse(filepath)
+except Exception as e:
+  print(e)
+
+filepath = "./config.json"
+
+try:
+  cfg = opentelemetry.Parse(filepath)
+except Exception as e:
+  raise e
+
+```
+
+#### Go Parse example
+
+```go
+
+filepath := "./config.yaml"
+cfg, err := otel.Parse(filepath)
+if err != nil {
+  return err
+}
+
+filepath := "./config.json"
+cfg, err := otel.Parse(filepath)
+if err != nil {
+  return err
+}
+
+```
+
+Implementations *MUST* allow users to specify an environment variable to set the configuration file. This gives flexibility to end users of implementations that do not support command line arguments. The proposed name for this variable:
+
+* `OTEL_CONFIG_FILE`
+
+The format for the configuration file will be detected using the file extension of this variable.
+
+### Configurer
+
+`Configurer` interprets a [Configuration model](#configuration-model) and produces configured SDK components.
+
+Multiple `Configurer`s can be [created](#createconfig---configurer) with different configurations. It is the caller's responsibility to ensure the [resulting SDK components](#get-tracerprovider-meterprovider-loggerprovider) are correctly wired into the application and instrumentation.
+
+`Configurer` **MAY** be extended in the future with functionality to apply an updated configuration model to the resulting SDK components.
+
+#### Create(config) -> Configurer
+
+Create a `Configurer` from a [configuration model](#configuration-model).
+
+#### Get TracerProvider, MeterProvider, LoggerProvider
+
+Interpret the [configuration model](#configuration-model) and return SDK TracerProvider, MeterProvider, LoggerProvider which strictly reflect the configuration object's details and ignores the [opentelemetry environment variable configuration scheme](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md).
+
+### Configuration model
+
+To allow SDKs and instrumentation libraries to accept configuration without having to implement the parsing logic, a `Configuration` model *MAY* be provided by implementations. This object:
+
+* has already been parsed from a file or data structure
+* is structurally valid (errors may yet occur when SDK or instrumentation interprets the object)
+
+### Configuration file
+
+The following demonstrates an example of a configuration file format (full example [here](https://github.com/MrAlias/otel-schema/blob/main/config.yaml)):
```

Comment:
You could consider adding this file as supplementary content to this PR (https://github.com/open-telemetry/oteps/tree/main/text/images or similar) for longevity.

Reply:
Added the config and schema examples to text/assets.